pkg/features/versioned_kube_features.go

```diff
@@ -486,7 +486,7 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 	},
 
 	LegacySidecarContainers: {
-		{Version: version.MustParse("1.0"), Default: true, PreRelease: featuregate.GA},
+		{Version: version.MustParse("1.0"), Default: false, PreRelease: featuregate.GA},
```

Comment:
We actually did use old code path before this feature, so it should be true.

Reply:
sorry let me look into this again. If it was on by default the test shouldn't be failing...